aion/web/elm/src/Auth/Views.elm

```diff
@@ -0,0 +1,149 @@
+
+
+
+-- login form section
+
+
+loginFormView : LoginForm -> Html Msg
+loginFormView loginForm =
+    div [ class "login-container" ]
+        [ h4 [] [ text "Log in:" ]
+        , Form.form []
+            [ emailLoginFormElement loginForm
+            , passwordLoginFormElement loginForm
+            , Button.button
+                [ Button.info
+                , Button.onClick Login
+                ]
+                [ text "submit" ]
+            ]
+        ]
+
+
+emailLoginFormElement : Forms.Form -> Html Msg
+emailLoginFormElement form =
+    Form.group []
+        [ Form.label [ for "email" ] [ text "Enter email below:" ]
+        , Input.text
+            [ Input.placeholder "john.doe@yahoo.com"
+            , Input.onInput (UpdateLoginForm "email")
+            , Input.value (Forms.formValue form "email")
+            ]
+        , Badge.pillInfo [] [ text (Forms.errorString form "email") ]
+        ]
+
+
+passwordLoginFormElement : Forms.Form -> Html Msg
+passwordLoginFormElement form =
+    Form.group []
+        [ Form.label [ for "password" ] [ text "Enter password below:" ]
+        , Input.password
+            [ Input.placeholder "Your password..."
+            , Input.onInput (UpdateLoginForm "password")
+            , Input.value (Forms.formValue form "password")
+            ]
+        , Badge.pillInfo [] [ text (Forms.errorString form "password") ]
+        ]
+
+
+
+-- registration form section
+
+
+registrationFormView : RegistrationForm -> Html Msg
+registrationFormView registrationForm =
+    div [ class "registration-container" ]
+        [ h4 [] [ text "Register:" ]
+        , Form.form []
+            [ nameRegisterFormElement registrationForm
+            , emailRegisterFormElement registrationForm
+            , passwordRegisterFormElement registrationForm
+            , Button.button
+                [ Button.info
+                , Button.onClick Register
+                ]
+                [ text "submit" ]
+            ]
+        ]
+
+
+nameRegisterFormElement : Forms.Form -> Html Msg
+nameRegisterFormElement form =
+    Form.group []
+        [ Form.label [ for "name" ] [ text "Enter username:" ]
+        , Input.text
+            [ Input.placeholder "John Doe"
+            , Input.onInput (UpdateRegistrationForm "name")
+            , Input.value (Forms.formValue form "name")
+            ]
+        , Badge.pillInfo [] [ text (Forms.errorString form "name") ]
+        ]
+
+
+emailRegisterFormElement : Forms.Form -> Html Msg
+emailRegisterFormElement form =
+    Form.group []
+        [ Form.label [ for "email" ] [ text "Enter email below:" ]
+        , Input.text
+            [ Input.placeholder "john.doe@yahoo.com"
+            , Input.onInput (UpdateRegistrationForm "email")
+            , Input.value (Forms.formValue form "email")
+            ]
+        , Badge.pillInfo [] [ text (Forms.errorString form "email") ]
+        ]
+
+
+passwordRegisterFormElement : Forms.Form -> Html Msg
```

Comment:
I think making forms too abstract will make it harder to make adjustments later. It may turn out that we need to change markup of one field and we are forced to refactor whole function and we are not flexible at all. I think having html markup at low abstraction level is good thing, especially in early development of this parts of code.

Reply:
Okay, we may leave this for the #56